### PR TITLE
/feature/Bocm 39 creation of endpoint patch

### PR DESCRIPTION
**Pull request by MariaValeroGonzalezKC**

### CODE REVIEW

File: apps/api/src/server/app/validation/validation.controller.js [hunk 1]
```diff
@@ -36,4 +37,14 @@ const getValidation = function (request, response) {
 	response.send(appealsList);
 };
 
-export { getValidation, getAppealReview };
+const updateValidation = function (request, response) {
+	console.log(request.text);
+	const errors = validationResult(request);
+	if (!errors.isEmpty()) {
+		console.log(errors);
```
Comment (mayya-bond-kc) [inline apps/api/src/server/app/validation/validation.controller.js:44]: In the future, I think we can remove these `console.log`

**Review by mayya-bond-kc — APPROVED** [review]
Looks good to me!

File: apps/api/src/server/app/validation/validation.controller.js (hunk 1 above)
Comment (MariaValeroGonzalezKC) [reply apps/api/src/server/app/validation/validation.controller.js:44]: will do in the next PR

**Pull request merged**